### PR TITLE
[stable/hubot] New Hubot chart

### PR DESCRIPTION
**Pull request by darioblanco**
**What this PR does / why we need it**: I wanted to have a way of provisioning hubot into my Kubernetes cluster with a Hubot 3 image we already open sourced and I think the community might benefit from this chart

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**: I haven't seen any hubot chart in this project yet and I would gladly take the role of maintaining a community one if we think it would be useful

### CODE REVIEW

File: stable/hubot/templates/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "hubot.fullname" . }}
+  namespace: {{ .Values.namespace }}
```
Comment (unguiculus) [inline stable/hubot/templates/configmap.yaml]: Remove namespaces from resources. That's unnecessary.

File: stable/hubot/templates/configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  labels:
+    app: {{ template "hubot.name" . }}
+    chart: {{ template "hubot.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+{{ toYaml .Values.hubot.config | indent 2 }}
+---
```
Comment (unguiculus) [inline stable/hubot/templates/configmap.yaml]: Separate files per resource, please.

File: stable/hubot/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+          livenessProbe:
+            httpGet:
+              path: /health
+              port: http
+          readinessProbe:
+            httpGet:
+              path: /health
+              port: http
+          volumeMounts:
+          - name: {{ template "hubot.fullname" . }}-scripts
```
Comment (unguiculus) [inline stable/hubot/templates/deployment.yaml]: Please use a consistent list indentation style throughout the chart. Either use zero or two spaces.

File: stable/hubot/values.yaml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+namespace: default
+
+replicaCount: 1
+
+image:
+  repository: minddocdev/hubot
+  tag: latest
```
Comment (unguiculus) [inline stable/hubot/values.yaml]: Use a stable version.

File: stable/hubot/values.yaml [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+  pullPolicy: Always
```
Comment (unguiculus) [inline stable/hubot/values.yaml]: IfNotPresent

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/hubot/templates/deployment.yaml [hunk 6]
```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "hubot.fullname" . }}
+  namespace: {{ .Values.namespace }}
```
Comment (unguiculus) [inline stable/hubot/templates/deployment.yaml]: Again, please remove namespaces from all resources.

File: stable/hubot/values.yaml [hunk 7]
```diff
@@ -0,0 +1,61 @@
+# Default values for hubot.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+namespace: default
```
Comment (unguiculus) [inline stable/hubot/values.yaml]: Remove namespace from `values.yaml`. You pass the namespace to the Helm command on installation.

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/hubot/templates/deployment.yaml (hunk 6 above)
Comment (darioblanco) [reply stable/hubot/templates/deployment.yaml]: Sorry, I totally overlooked that. Thanks for reminding 👍

**Review by unguiculus — APPROVED** [review]

**Pull request merged**